cmd/root.go

```diff
@@ -52,12 +49,44 @@ then starts your system shell with shortcuts configured for wash subcommands.`,
 		Args:          cobra.MaximumNArgs(1),
 		Version:       version,
 	}
-	addServerArgs(rootCmd, "warn")
-	// rootCommandFlag is used in rootMain.go.
-	rootCmd.Flags().StringVarP(&rootCommandFlag, "command", "c", "", "Run the supplied string and exit")
+
+	if config.Embedded {
+		rootCmd.Use = ""
+		rootCmd.PreRun = nil
+		rootCmd.Long = ""
+		// Augment the usage template to minimize usage when set to empty.
+		rootCmd.SetUsageTemplate(`Usage:{{if (and .Runnable (ne .Use ""))}}
```

Comment:
Was this copy-pasted from Cobra? Just curious.

Reply:
Yeah, then tweaked slightly.

Reply:
Ok. May be nice to extract it into a variable b/c it is a bit long.